blockchain/ethereum/contracts/erc1155/erc1155presetminterpauser abi/0b58f32d-0aac-454c-bf10-cbf22fabc5fa.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "blockchain/ethereum/contracts/erc1155/erc1155presetminterpauser abi",
```

Comment:
Should the name be camelcased? like "blockchain/ethereum/contracts/erc1155/ERC1155PresetMinterPauser ABI"